superset-frontend/src/views/CRUD/chart/ChartCard.tsx

```diff
@@ -102,22 +102,26 @@ export default function ChartCard({
         </Menu.Item>
       )}
       {canExport && (
-        <Menu.Item
-          role="button"
-          tabIndex={0}
-          onClick={() => handleBulkChartExport([chart])}
-        >
-          <ListViewCard.MenuIcon name="share" /> {t('Export')}
+        <Menu.Item>
+          <div
```

Comment:
Curious why this change is needed, only in case it's an indication of things we'll have to do elsewhere.

Reply:
@rusackas Antd adds a span around the SVG. There might be cases where related styles affect the span causing the Icon to behave incorrectly. This was one of those cases. As a container DIV was used elsewhere in this file, I went ahead and made it consistent everywhere. By doing so the style override issue gets resolved as well.